documentation/docs/steps/cloudFoundryDeleteService.md

```diff
@@ -4,7 +4,9 @@
 
 ## Prerequisites
 
-This step is for deleting an exisiting service on Cloud Foundry
+This step is for deleting an exisiting service on Cloud Foundry.
+You need to provide the Cloud Foundry API Endpoint, the Organisation as well as the space and the respective Service instance name you want to delete. 
```

Comment:
Could you make the capitalization consistent? So either lowercase (endpoint, organisation, space, service instance name) or uppercase.